spec/models/variant_override_spec.rb

```diff
@@ -54,7 +54,7 @@
           let(:count_on_hand) { nil }
 
           it "is valid" do
-            expect(variant_override.save).to be_truthy
+            expect(variant_override).to be_valid
```

Comment:
interesting. I wonder what other devs think about this. Shall we make this the standard to test models? no saving, just valid. Is this not a slightly less complete test?

Reply:
I am ok either way, I think we should make one of these a convention.

Reply:
I think it looks good in this case. The test is to see if the object is valid...

Reply:
my question was more: is testing that it is valid a good test, or should we test that it can be saved correctly?
maybe the answer is, it doesn't really matter, it's not a big difference.

Reply:
> The test is to see if the object is valid...

Exactly!

Reply:
> should we test that it can be saved correctly?

In that case would `#save!` be better than `#save`?

Reply:
yes, I mean save!

Reply:
well, let's move one!